test/test_request.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+
+from urllib3 import request
+
+from dummyserver.testcase import HTTPDummyServerTestCase
+
+
+class TestRequest(HTTPDummyServerTestCase):
+    @classmethod
```

Comment:
I don't think we need the `setup_method`

Reply:
Done